app/pages/reset-password/new-password-form.spec.js

```diff
@@ -5,15 +5,41 @@ import sinon from 'sinon';
 import NewPasswordForm from './new-password-form';
 
 describe('NewPasswordForm', () => {
+  let wrapper;
+
+  before(() => {
+    wrapper = shallow(<NewPasswordForm />);
+  });
+
   it('renders a new password form for a user', () => {
-    const wrapper = shallow(<NewPasswordForm />);
     assert.equal(wrapper.find('form').length, 1);
   });
 
   it('should call onSubmit when user submits a new password', () => {
     const newPasswordSpy = sinon.spy();
-    const wrapper = mount(<NewPasswordForm onSubmit={newPasswordSpy} />);
+    wrapper = mount(<NewPasswordForm onSubmit={newPasswordSpy} />);
     wrapper.find('button').simulate('submit');
```

Comment:
The `onSubmit` handler is on the form, not the button, so you'll want to `wrapper.find('form').simulate('submit').